### PR TITLE
Fix/parse configs ambiguously

### PR DESCRIPTION
**Pull request by Mropat**
This PR adds/fixes ...

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
- [x] activate stage: `us`
- [x] request trailblazer-stage on hasta: `paxa`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh [THIS-BRANCH-NAME]`
- [x] ssh to clinical-db (depending on type of change)
- [x] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### Expected test outcome
- [x] check that trailblazer operates exactly the same on stage, but also can track fluffy
- [x] Take a screenshot and attach or copy/paste the output.
image

## Review
- [x] code approved by HS
- [x] tests executed by MR
- [x] "Merge and deploy" approved by MR
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: trailblazer/store/api.py [hunk 1]
```diff
@@ -246,8 +246,8 @@ def query_slurm(job_id_file: str, case_id: str, ssh: bool) -> Any:
         case_id: Unique internal case identifier which is expected to by the only item in the .YAML dict
         ssh : Whether the request is executed from hasta or clinical-db"""
         job_id_dict = safe_load(open(job_id_file))
-        submitted_jobs = job_id_dict[case_id]
-        jobs_string = ",".join(submitted_jobs)
+        submitted_jobs = job_id_dict.get(next(iter(job_id_dict)))
+        jobs_string = ",".join(map(str, submitted_jobs))
```
Comment (henrikstranneheim) [inline trailblazer/store/api.py:250]:
```suggestion
        submitted_job_ids = job_id_dict.get(next(iter(job_id_dict)))
        job_ids_string = ",".join(map(str, submitted_job_ids))
```

Comment (henrikstranneheim) [reply trailblazer/store/api.py:250]: I think you should put L248-250 in a "get_submitted_job_ids" def in perhaps a `proccess` or `slurm` module or similar. It does not feel like this processing belongs in store/api and also you would get the benefit of writing a unit test to check this parsing. Also, the hardcoded input to check_ouput() looks like it belongs in a slurm_api. What do you think? Perhaps out of scope for this PR.

File: .github/workflows/test_and_coverage.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Test with pytest & Coveralls
       run: |
         pytest --cov=trailblazer/
-        coveralls
+        coveralls --service=github
```
Comment (henrikstranneheim) [inline .github/workflows/test_and_coverage.yml:35]: Was that the cause - 👍

**Review by henrikstranneheim — APPROVED** [review]
Nice!

File: .github/workflows/test_and_coverage.yml (hunk 2 above)
Comment (Mropat) [reply .github/workflows/test_and_coverage.yml:35]: Yup! thanks @northwestwitch :D

File: trailblazer/store/api.py (hunk 1 above)
Comment (Mropat) [reply trailblazer/store/api.py]: Sure! This could be something for gRPC-based Trailblazer v.3 :D

**Comment by Mropat** [comment]
Deployed on prod

```
Successfully installed trailblazer-9.2.1
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/maryia.ropart
done.
Log deploy... done.
Running --version on each Clinical-Genomics developed tool to verify that all requirements are met:
argcomplete 1.12.1 has requirement importlib-metadata<3,>=0.23, but you have importlib-metadata 0.8.
cg, version 18.3.0
housekeeper, version 3.2.0
demux, version 5.2.2
cglims, version 1.3.4

```

**Pull request merged**